app/Scripts/BoidAbsorptionComponent.gd

```diff
@@ -0,0 +1,23 @@
+extends Node
+
+onready var _enemy_manager = get_node("/root/Main/EnemyManager")
+
+# Declare member variables here. Examples:
+# var a = 2
+# var b = "text"
+var _player: Node 
+
+
+func _on_enemy_death() -> void:
+	_player.add_boid()
+	
+
+# Called when the node enters the scene tree for the first time.
+func _ready():
+	_player = get_parent()
+	_enemy_manager.subscribe_to_deaths(funcref(self, "_on_enemy_death"))
```

Comment:
funcref allows us to create references to functions. This is used to implement a subscription pattern